portfolio/pom.xml

```diff
@@ -60,7 +96,8 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <deploy.projectId>PROJECT_ID_HERE</deploy.projectId>
+          <!-- TODO: set project ID. -->
+          <deploy.projectId>olabodeige-step-2020</deploy.projectId>
```

Comment:
Recommend changing this to PROJECT_ID_HERE so that others know to change it when they deploy.